tests/settings/SectionFillingTest.py

```diff
@@ -87,6 +89,18 @@ def test_fill_section(self):
         self.assertEqual(new_section['key'].value, 'val')
         self.assertEqual(len(new_section.contents), 3)
 
+        with LogCapture() as capture:
+            fill_section(self.test_section,
+                         lambda param_1, param_2: {},
+                         self.log_printer,
+                         [LocalTestBear, GlobalTestBear])
+            capture.check(
+                ('root',
+                 'WARNING',
+                 'acquire_settings: section parameter '
+                 'is deprecated.')
```

Comment:
maybe this one fits into a single line, makes it a bit easier to read ;)

Reply:
ok will do it